### PR TITLE
(#3755) Handle missed order data

### PR DESCRIPTION
**Pull request by IrynaMlshvk**
#3755

### CODE REVIEW

File: includes/admin/settings/class-wc-admin-settings-woorule.php [hunk 1]
```diff
@@ -124,7 +123,6 @@ public static function order_status_changed($id, $status = '', $new_status = '')
                           'subtotal' => $item->get_total()
                           );
 
-                        // this is bullshit
```
Comment (IlyaKobus) [inline includes/admin/settings/class-wc-admin-settings-woorule.php:127]: is it stopped to be a bullshit? 😁

**Review by IlyaKobus — APPROVED** [review]

File: includes/admin/settings/class-wc-admin-settings-woorule.php (hunk 1 above)
Comment (IrynaMlshvk) [reply includes/admin/settings/class-wc-admin-settings-woorule.php:127]: @IlyaKobus I think it was related to another part of code that was changed with time 😄

File: includes/admin/settings/class-wc-admin-settings-woorule.php [hunk 2]
```diff
@@ -146,7 +144,6 @@ public static function order_status_changed($id, $status = '', $new_status = '')
                     }
 
                     $order_data = $order->get_data();
-                    $phone = $order->get_billing_phone();
```
Comment (dmitry-rule) [inline includes/admin/settings/class-wc-admin-settings-woorule.php:149]: It was extra (unneeded) data right? *and phone will be taken from $order_data['billing']['phone']

Comment (IrynaMlshvk) [reply includes/admin/settings/class-wc-admin-settings-woorule.php:149]: @dmitry-rule yes, it was unused field

**Pull request merged**